userappwithbroker/userapp-rules.pro

```diff
@@ -0,0 +1,66 @@
+# Add project specific ProGuard rules here.
```

Comment:
Do we need this? (given that user app is our internal test app... not having this means one less thing to maintain.)

Reply:
I think we should dog food the ProGuard rules.